addroom.html

```diff
@@ -17,52 +18,126 @@
         margin: 0;
         padding: 0;
       }
+
+       /*set the background*/
+      .title{
+        padding:100px;
+        margin-top:-30px;
+      } 
+      .center{text-align:center}
+      .textBg{
+        background:rgb(209, 206, 206);
+        height:500px; 
+        width:500px;
+        text-align: center;
+        margin-left: 375px;
+        margin-top:-50px;
+        margin-bottom: 100px;
+        font-size:100%;
+        color:rgb(17, 17, 17);
+        border-radius:25px;
+        padding:0.5cm;
+      }
+      /*set the nav bar*/
+      ul {
+       list-style-type: none;
+       margin: 0;
+       padding: 0;
+       overflow: hidden;
+       background-color:#e2e7ec;
+       height:60px;
+      }
+
+      li {
+       float: left;
+      }
+
+      li a {
+       display: block;
+       color: rgb(17, 16, 16);
+       text-align: center;
+       padding: 15px 20px;
+       text-decoration: none;
+      }
+
+      li a:hover {
+       background-color: #e2e7ec;
+      }
+      
+      a.leftPand{
+       margin-left:-80px;
+      }
+
     </style>
     <link href="css/bootstrap.min.css" rel="stylesheet" />
     <link href="css/style.css" rel="stylesheet" />
   </head>
+  
+  
+  
   <body>
-    <div id="map"></div>
+      <ul>
+          <li><img  src="logo.png" width="40%" height="40%"><li>
+          <li><a class="" href="#home">Home</a></li>
+          <li><a href="file:///Users/ericguochn/Downloads/kitchen-towel-master%202/signup.html#news">Sing Up</a></li>
+          <li><a href="file:///Users/ericguochn/Downloads/kitchen-towel-master%202/index.html">Sing In</a></li>
```

Comment:
Hyperlinks should be relative to the domain name, such as "/index.html", not the path on your own computer.

Also it should be "Sign up" and "Sign in", not "Sing".